tests/testOpExportSlot.py

```diff
@@ -79,8 +79,8 @@ def testBasic_Hdf5(self):
             opRead.cleanUp()
 
     def testBasic_Npy(self):
-        data = numpy.random.random( (100,100) ).astype( numpy.float32 )
-        data = vigra.taggedView( data, vigra.defaultAxistags('xy') )
+        data = numpy.random.random( (100,100,1) ).astype( numpy.float32 )
```

Comment:
please add a space after the comma, please do so also for the other changed line with the `CoordinateOffset`